README.md

```diff
@@ -1,24 +1,27 @@
-# README
-
-This README would normally document whatever steps are necessary to get the
-application up and running.
-
-Things you may want to cover:
-
-* Ruby version
-
-* System dependencies
-
-* Configuration
-
-* Database creation
-
-* Database initialization
-
-* How to run the test suite
-
-* Services (job queues, cache servers, search engines, etc.)
-
-* Deployment instructions
-
-* ...
+#Programming Learning Journal 
+ This is a Ruby on Rails 5 application that will help you keep track of your progress as you gain experience with a variety of languages, libraries, frameworks and technologies.
+
+##Getting Started
+To get the Rails server running locally:
+
+*Clone this repo
+*bundle install to install all req'd dependencies
+*rake db:migrate to make all database migrations
+*rails s to start the local server
+
+##Dependencies
```

Comment:
I think dependencies would be better served explaining what a developer would need to be running on their PC to run the app. For example, do you remember you had an issue because you didn't have `nodejs`? Common dependencies are the database (`sqlite3` in dev for your app, IIRC), Ruby itself, Rails and node. That's all off the top of my head.

Reply:
I created two sections now. One for the System dependencies and another one for the Gem dependencies.

Reply:
I think you solved the problem really nicely with your Prerequisite section, don't worry about creating system dependencies too as there will be some duplication.